ui/app/AppLayouts/Profile/helpers/ProfileShowcaseDirtyState.qml

```diff
@@ -31,6 +31,15 @@ QObject {
       */
     readonly property alias hiddenModel: hidden
 
+    /**
+      * Returns dirty state of the showcase model.
+      */
+    readonly property alias dirty: writable.dirty
+
+    function revert() {
```

Comment:
Is this to discard changes when user pressess "cancel" button in the dirty bubble??